### PR TITLE
Add unit test for func matchesElbTags

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@cn.fujitsu.com>

### CODE REVIEW

File: pkg/resources/aws/aws_test.go [hunk 1]
```diff
@@ -196,3 +197,71 @@ func TestSharedVolume(t *testing.T) {
...
+
+	for i, test := range tc {
+		got := matchesElbTags(test.tags, test.actual)
+		if got != test.expected {
+			t.Fatalf("unexpected result from testcase %d, expected %v, got %v", i, test.expected, got)
+		}
+	}
+}
+
+func strtopoint(str string) *string {
```
Comment (olemarkus) [inline pkg/resources/aws/aws_test.go:259]: We have `fi.String` that does the same thing.

**Comment by hs0210** [comment]
@olemarkus I have replaced the func strtopoint with fi.String. PTAL.

**Comment by olemarkus** [comment]
Can you fix te bazel test? probably enough to run `hack/update-bazel.sh`
Looks good otherwise.

File: pkg/resources/aws/aws_test.go [hunk 2]
```diff
@@ -196,3 +198,67 @@ func TestSharedVolume(t *testing.T) {
...
 }
+
+func TestMatchesElbTags(t *testing.T) {
+	tc := []struct {
+		tags     map[string]string
+		actual   []*elb.Tag
+		expected bool
+	}{
+		{
+			tags: strtomap("tagkey1", "tagvalue1"),
```
Comment (hakman) [inline pkg/resources/aws/aws_test.go:209]: Why use `strtomap` when you could directly create a map?
```suggestion
			tags: map[string]string{"tagkey1": "tagvalue1"}
```

**Comment by hs0210** [comment]
@hakman Thanks.

**Comment by hs0210** [comment]
@olemarkus Sorry, can you explain a little more, I am not very clear what I need to fix.

**Comment by hakman** [comment]
Happy to help @hs0210. Could you also add some tests for multiple tags? The `matchesElbTags` function should handle those also. Thanks!

@olemarkus asked that you run `hack/update-bazel.sh` and commit the changes to fix the 2 failing tests.

**Comment by hs0210** [comment]
@hakman I see these tags are all in string format, don't they?

**Comment by hs0210** [comment]
PTAL.

**Pull request merged**